src/main/java/org/rulez/demokracia/pdengine/ContainingBeats.java

```diff
@@ -0,0 +1,50 @@
+package org.rulez.demokracia.pdengine;
+
+import org.rulez.demokracia.pdengine.BeatTable.Direction;
+import org.rulez.demokracia.pdengine.dataobjects.Pair;
+import org.rulez.demokracia.pdengine.exception.ReportedException;
+import org.rulez.demokracia.types.Matrix;
+
+public interface ContainingBeats extends Matrix<Choice, Pair>{
+	
+	long serialVersionUID = 1L;
```

Comment:
is this needed here, in an interface?

Reply:
If the interface does not contain this id the Ci build fail. So i dont know what to do. I know this is not the nice solution.

Reply:
strange... then leave it here.